python-mongo/infra/shared/apps-env.bicep

```diff
@@ -0,0 +1,33 @@
+param name string
```

Comment:
The idea behind the `shared` folder is that these are individual resources shared between services. They are not meant to be library components, but rather mapping to shared resources similar to the `app` folder.